homework_04.py

```diff
@@ -0,0 +1,119 @@
+"""big variable."""
+
+adwentures_of_tom_sawer = """\
+Tom gave up the brush with reluctance in his .... face but alacrity
+in his heart. And while
+the late steamer
+"Big Missouri" worked ....
+and sweated
+in the sun,
+the retired artist sat on a barrel in the .... shade close by, dangled his legs,
+munched his apple, and planned the slaughter of more innocents.
+There was no lack of material;
+boys happened along every little while;
+they came to jeer, but .... remained to whitewash. ....
+By the time Ben was fagged out, Tom had traded the next chance to Billy Fisher
+ for
+a kite, in good repair;
+and when he played
+out, Johnny Miller bought
+in for a dead rat and a string to swing it with—and so on, and so on,
+hour after hour. And when the middle of the afternoon came, from being a
+poor poverty, stricken boy in the .... morning, Tom was literally
+rolling in wealth."""
+
+# ПЕРЕЗАПИСУЙТЕ зміст змінної adwentures_of_tom_sawer у завданнях 1-3
+# task 01 ==
+""" Дані у строці adwentures_of_tom_sawer розбиті випадковим чином,
+через помилку.
+треба замінити кінець абзацу на пробіл .replace("\n", " ")"""
+
+"""replace newline characters with spaces"""
+
+tom_space = adwentures_of_tom_sawer.replace('\n', ' ')
+
+# task 02 ==
+""" Замініть .... на пробіл
+"""
+"""replace '....' with spaces"""
+
+tom_dot = tom_space.replace('....', ' ')
+
+# task 03 ==
+""" Зробіть так, щоб у тексті було не більше одного пробілу між словами.
+"""
+
+"""only one space between words"""
+
+tom_final = tom_dot.replace('   ', ' ')
+
+print(tom_final)
+
+# task 04
+""" Виведіть, скількі разів у тексті зустрічається літера "h"
+"""
+
+"""count of letter 'h'"""
+
+tom_h = adwentures_of_tom_sawer.count('h')
+
+print(tom_h)
+
+# task 05
+""" Виведіть, скільки слів у тексті починається з Великої літери?
+"""
+
+"""count words starting with uppercase"""
+
+tom_upper = [upper for upper in adwentures_of_tom_sawer if upper.isupper()]
+
+print('Кількість великих літер:', len(tom_upper))
+
+# task 06
+""" Виведіть позицію, на якій слово Tom зустрічається вдруге
+"""
+"""find the position of the second 'Tom'"""
+
+tom_first = adwentures_of_tom_sawer.find('Tom')
+
+tom_second = adwentures_of_tom_sawer.find('Tom', tom_first + 1)
+
+print(tom_second)
+
+# task 07
+""" Розділіть змінну adwentures_of_tom_sawer по кінцю речення.
+Збережіть результат у змінній adwentures_of_tom_sawer_sentences
+"""
+"""split text by sentence"""
+adwentures_of_tom_sawer_sentences = tom_final.split('.')
```

Comment:
split('. ')  Better to add space after period

Reply:
added